src/bower-utils.js

```diff
@@ -76,6 +76,12 @@ module.exports = {
               'repository'
             ]);
 
+            // scoped packages get special treatment
+            // See https://github.com/npm/npm/blob/2a5977e0c65b244e92d848fcd56f2f80ba8cdf3b/lib/pack.js#L53
```

Comment:
I'd rather link to tag instead of a sha1 to be sure the link will remain valid (as long as the tag is not deleted, which should not happen!).